pkg/provisioners/heartbeats/config/examples/source.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Source
+metadata:
+  name: love
+  namespace: default
+spec:
+  provisioner:
+    ref:
+      name: heartbeats
+  arguments:
+    label: "<3"
+    period: 2
```

Comment:
Could the `kind: Source` also reference to an `image`, backing the code (if not using `ko` )?

Reply:
Did this. I intended to learn something with the heartbeat source and I have... :D